impl/src/test/java/org/ehcache/GettingStarted.java

```diff
@@ -69,6 +73,27 @@ public void testWikiExample() {
   }
 
   @Test
+  public void testTieredStore() throws Exception {
+    CacheConfiguration<Long, String> tieredCacheConfiguration = newCacheConfigurationBuilder()
+        .persistenceMode(CacheConfiguration.PersistenceMode.SWAP)
+        .withResourcePools(newResourcePoolsBuilder().with("heap").with("disk").build())
```

Comment:
Would use the following names:
`.withCapacityConstraints(newCapacityConstraintsBuilder().heap(10).disk(100).build())`